pkgs/top-level/all-packages.nix

```diff
@@ -11586,7 +11586,8 @@ in
   inherit (beam.interpreters)
     erlang erlangR23 erlangR22 erlangR21 erlangR20 erlangR19 erlangR18
     erlang_odbc erlang_javac erlang_odbc_javac erlang_basho_R16B02
-    elixir elixir_1_11 elixir_1_10 elixir_1_9 elixir_1_8 elixir_1_7;
+    elixir elixir_1_11 elixir_1_10 elixir_1_9 elixir_1_8 elixir_1_7
+    elixir_ls;
```

Comment:
This should be done automatically for all packages because we don't want to maintain a list of packages here.

Reply:
I've added the suggested fix. Since it was adding some other packages on top, I've removed those.
Waiting for the build to check.

Reply:
I ran into an interesting problem here.
You can recursively add packages, that was not a problem.
The solution looked something like
```
beam = (let
   packages = callPackage ./beam-packages.nix { };
   interpreters = packages.interpreters;
in 
   packages // removeAttrs interpreters [ "lfe" "lfe_1_2" "lfe_1_3" ]);
```

However you still need to inherit those. I haven't found a way to recursively inherit something.
So unless I am missing something, we need to keep the current syntax

Reply:
I think you need to use passthru in the packages for those attributes.

Reply:
I have opened a separate issue for this
https://github.com/NixOS/nixpkgs/issues/119201
just so that it doesn't block this PR.